<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>722</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>15</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
font: 700 italic 15pt &quot;Ubuntu&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>341</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>505</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_key">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ключ&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_val">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_value">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Строка текста&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditText"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_add_item">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Добавить элемент</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_del">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Удалить по ключу:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_delValue">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bnt_remove">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="lbl_find">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Найти по ключу:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_findValue">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bnt_search">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>Найти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ключ 1:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="el_1">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ключ 2:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="el_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_cnt_of_list">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести наименьший ключ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_pre">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести ключи с начала</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_post">
       <property name="minimumSize">
        <size>
         <width>438</width>
         <height>31</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести ключи с конца</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_in">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести ключи в порядке возрастания</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_add_subtree">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Добавить поддерево</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_remove_subtree">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Удалить поддерево</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTreeView" name="treeView_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>500</y>
      <width>341</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>470</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Sans</family>
      <pointsize>14</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Поддерево:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>520</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="lbl_key_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>Ключ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spb_val_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="lbl_value_2">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
         </property>
         <property name="text">
          <string>Строка текста</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditText_2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="bnt_add_item_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Ubuntu Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Добавить элемент</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
